sci-biology/salmon/salmon-0.5.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
It's fine for now but jlec deprecated EAPI 5. Doing stuff from a prefix is indeed problematic. You are lucky if `repoman` works at all there (personal experience). Just don't forget we are in 2016.

I don't have merge power anymore (I probably have on github but I think I lost the privilege to do it on the official repo) so someone else will have to do it.